### DISCUSSION TITLE
Android 8 (Oreo)

### DISCUSSION THREAD

**Discussion by marciorp-br** [standardnotes/forum#3545] [General]
Is it possible to compile the Android app compatible with version 8 (Oreo)?

Very thanks!

**Comment by ghost** [comment]
😗nice idea, try it.

**Comment by marciorp-br** [comment]
> 😗nice idea, try it.

Unfortunately I have no idea how to do this. I'm not an Android developer.

**Comment by ghost** [comment]
😀 don't worry, I'll try it
